server/message_hooks.go

```diff
@@ -97,6 +97,10 @@ func (p *Plugin) MessageHasBeenPosted(_ *plugin.Context, post *model.Post) {
 		return
 	}
 
+	if post.UserId == p.GetBotUserID() {
+		return
+	}
+
```

Comment:
Fascinating: is this an extra check, or was this literally something we didn't previously catch?

Reply:
I could not find a check in the code before, and I believe this is the cause of this issue: https://mattermost.atlassian.net/browse/MM-56022